Cake.Issues.Recipe/Content/tasks/buildservers/GitHubActionsBuildServer.cake

```diff
@@ -128,4 +136,47 @@ public class GitHubActionsBuildServer : BaseBuildServer
             context.Warning($"Failed to upload SARIF report to GitHub code scanning. Status: {response.StatusCode}, Error: {errorContent}");
         }
     }
+
+    private static bool IsCodeScanningEnabled(ICakeContext context, string repository, string token)
+    {
+        // Check if code scanning is enabled by attempting to fetch code scanning alerts
+        var apiUrl = new System.Uri($"https://api.github.com/repos/{repository}/code-scanning/alerts?per_page=1");
```

Comment:
Doesn't Octokit provide an API for this?

We already have Octokit added to the project as a transitive dependency of the Cake.GitHib addin.